onnxruntime/core/common/threadpool.cc

```diff
@@ -307,14 +306,14 @@ class alignas(CACHE_LINE_BYTES) LoopCounter {
   bool ClaimIterations(unsigned my_home_shard,
                        unsigned& my_shard,
                        uint64_t& my_start,
-                       uint64_t& my_end) {
+                       uint64_t& my_end, uint64_t block_size) {
```

Comment:
```suggestion
                       uint64_t& my_end, 
                       uint64_t block_size) {
```

[ adjusting indentation to match surrounding items ]